README.md

```diff
@@ -87,14 +87,24 @@ We'll start by creating the workflow file to publish a Docker image to GitHub Pa
        steps:
          - uses: actions/checkout@v3
          # Add your test steps here if needed...
-         - name: Build container
-           uses: docker/build-push-action@v1
+         - name: Docker meta
```

Comment:
@sinsukehlab Can you explain why this part is needed? Rather than just updating `docker/build-push-action` to v3?

Reply:
Certainly. The Boolean env `tag_with_sha` is abolished in docker/build-push-action@v2 or higher. I use docker/metadata-action to handle tags.
See: https://docs.docker.com/build/ci/github-actions/examples/#manage-tags-and-labels

Reply:
The link is now https://docs.docker.com/build/ci/github-actions/manage-tags-labels/